tools/gen-vast-slices/README.md

```diff
@@ -0,0 +1,23 @@
+# gen-vast-slices
+
+The **gen-vast-slices** tool parses supported log formats from STDIN into table
+slices and prints them in serialized form to STDOUT.
```

Comment:
```suggestion
slices and prints them in serialized, *binary* form to STDOUT.
```

I would also add that the binary format comes from `caf::binary_serializer`.

Reply:
That reads like the output to STDOUT is binary, which isn't the case.

Reply:
I took a slightly different wording, hope that's a good compromise.